components/ILIAS/OnScreenChat/classes/class.ilOnScreenChatGUI.php

```diff
@@ -288,6 +308,9 @@ public static function initializeFrontend(ilGlobalTemplateInterface $page): void
             iljQueryUtil::initjQueryUI($page);
             ilLinkifyUtil::initLinkify($page);
 
+            $page->addJavaScript('assets/js/modal.js');
+            $page->addJavaScript('assets/js/socket.io.min.js');
+            $page->addJavaScript('assets/js/Chatroom.min.js');
```

Comment:
Do we really want to include the repository chat JS file on every page just to retrieve the "Bus" module?

Reply:
It's not just the bus. It's also to send ajax request and for autocompletion